packages/base/src/config/ThemeRoot.ts

```diff
@@ -0,0 +1,74 @@
+import createLinkInHead from "../util/createLinkInHead.js";
```

Comment:
**Note:** `ThemeRoot.ts` is not a new file. It used to be named `ThemeRoots.ts` (GitHub does not provide diff).